CONTRIBUTING.rst

```diff
@@ -10,7 +12,44 @@ developed primarily by a group of U.S. Department of Energy subcontractors, our
 goal is to increase community involvement and integration over time. We welcome
 your contributions!
 
-.. note:: This document is a work in progress.
+
+Copyright
+---------
+
+All submissions to the Celeritas project are automatically licensed under the
+terms of :ref:`the project copyright <code_copyright>` as formalized by the
+`GitHub terms of service`_.
+
+.. _GitHub terms of service: https://docs.github.com/en/github/site-policy/github-terms-of-service#6-contributions-under-repository-license
+
+
+Collaborating
+-------------
+
+Working with other Celeritas team members is a critical part of the development
+process. Please join the ``#code`` channel on the `Celeritas Slack workspace`_,
+open an issue with a bug report or feature request, or start a discussion on
+GitHub.
+
+.. _Celeritas Slack workspace: https://celeritasproject.slack.com/
+
+
+Development prerequisites
+-------------------------
+
+To meet the :ref:`formatting` requirements described in the development guide,
+make sure that `clang-format`_ is installed on your development
```

Comment:
Do we keep anywhere (else) the minimal version of clang-format to use?

Reply:
Good point. Generally they produce nearly identical results but maybe we need to specify... although I hesitate to add it to the documentation since it means we'll have to remember to update it.

I also am thinking about changing the current settings
```
AlignConsecutiveAssignments: true
AlignConsecutiveDeclarations: true
```
to false (for declarations or for both), since these cause the most changes from one clang-format version to the next and the most inconsistencies when formatting only a patch.

Reply:
This is a tough call.  It would make the code a bit more messy to look at but indeed would be more stable across clang-format version.  The main question is whether this is stabilizing or if new/upcoming version of clang-format are still struggling with that.